pkg/dbutil/common.go

```diff
@@ -118,6 +119,10 @@ func OpenDB(cfg DBConfig) (*sql.DB, error) {
 		dbDSN = fmt.Sprintf("%s:%s@tcp(%s:%d)/?charset=utf8mb4", cfg.User, cfg.Password, cfg.Host, cfg.Port)
 	}
 
+	for key, val := range vars {
+		dbDSN += fmt.Sprintf("&%s='%s'", key, url.QueryEscape(val))
```

Comment:
seems whole value including `'` should be URLescaped 

https://github.com/go-sql-driver/mysql#system-variables